interfaces/builtin/unity8.go

```diff
@@ -20,11 +20,66 @@
 package builtin
 
 import (
+	"bytes"
 	"fmt"
 
 	"github.com/snapcore/snapd/interfaces"
 )
 
+const unity8ConnectedPlugAppArmor = `
+# Description: Can access unity8 desktop services
+
+#include <abstractions/dbus-session-strict>
+
+# Fonts
+#include <abstractions/fonts>
+/var/cache/fontconfig/   r,
+/var/cache/fontconfig/** mr,
+
+# The snapcraft desktop part may look for schema files in various locations, so
+# allow reading system installed schemas.
+/usr/share/glib*/schemas/{,*}              r,
+/usr/share/gnome/glib*/schemas/{,*}        r,
+/usr/share/ubuntu/glib*/schemas/{,*}       r,
+
+# URL dispatcher. All apps can call this since:
+# a) the dispatched application is launched out of process and not
+#    controllable except via the specified URL
+# b) the list of url types is strictly controlled
+# c) the dispatched application will launch in the foreground over the
+#    confined app
+dbus (send)
+     bus=session
+     path=/com/canonical/URLDispatcher
+     interface=com.canonical.URLDispatcher
+     member=DispatchURL
+     peer=(name=com.canonical.URLDispatcher,label=###SLOT_SECURITY_TAGS###),
+
+# Note: content-hub may become its own interface, but for now include it here
+# Pasteboard via Content Hub. Unity8 with mir has safeguards that ensure snaps
+# only may get/set the pasteboard with user-driven actions.
+dbus (send)
+     bus=session
+     interface=com.ubuntu.content.dbus.Service
+     path=/
+     member={CreatePaste,GetAllPasteIds,GetLatestPasteData,GetPasteData,GetPasteSource,PasteFormats,RequestPasteByAppId,SelectPasteForAppId,SelectPasteForAppIdCancelled}
+     peer=(name=com.ubuntu.content.dbus.Service,label=###SLOT_SECURITY_TAGS###),
+dbus (receive)
+     bus=session
+     interface=com.ubuntu.content.dbus.Service
+     path=/
+     member={PasteboardChanged,PasteFormatsChanged,PasteSelected,PasteSelectionCancelled}
+     peer=(name=com.ubuntu.content.dbus.Service,label=###SLOT_SECURITY_TAGS###),
+
+# Lttng tracing is very noisy and should not be allowed by confined apps.
+# Can safely deny. LP: #1260491
+deny /{dev,run,var/run}/shm/lttng-ust-* r,
```

Comment:
Can you make this explicit deny this instead:
```
deny /{dev,run,var/run}/shm/lttng-ust-* rw,
```

I noticed this while looking at kiosk issues this week.